hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1608,9 +1606,31 @@
               - lc_seed_hw_rd_en_o: impacts flash ctrl & OTP ctrl
             - These outputs are enabled per the [life cycle architecture spec]({{< relref "doc/security/specs/device_life_cycle/#architecture" >}}).
             X-ref'ed with the respective IP tests that consume these signals.
-            '''
-      stage: V2
-      tests: []
+
+            Note that the following signals are already verified with connectivity tests and SVAs:
+              - lc_dft_en_o (AST connection)
+              - lc_cpu_en_o
+              - lc_escalate_en_o
+            '''
+      stage: V2
+      tests: [
+        "chip_prim_tl_access",                    // lc_dft_en_o: otp_ctrl, flash_ctrl
+        "chip_tap_straps_dev",                    // lc_dft_en_o, lc_hw_debug_en_o: pinmux
+        "chip_tap_straps_prod",                   // lc_dft_en_o, lc_hw_debug_en_o: pinmux
+        "chip_tap_straps_rma",                    // lc_dft_en_o, lc_hw_debug_en_o: pinmux
+        // TODO                                   // lc_nvm_debug_en_o: flash_ctrl
+        "chip_sw_clkmgr_external_clk_src_for_sw", // lc_hw_debug_en_o: clkmgr
+        "chip_sw_sram_ctrl_execution_main",       // lc_hw_debug_en_o: sram_ctrl main
+        // TODO(?)                                // lc_hw_debug_en_o: sram_ctrl ret
+        // TODO: ref chip_sw_rv_dm_lc_disabled    // lc_hw_debug_en_o: rv_dm
+        "chip_sw_keymgr_key_derivation",          // lc_keymgr_en_o: keymgr
+        // TODO: add disable test                 // lc_keymgr_en_o: keymgr
```

Comment:
might be good to make this its own issue.

Reply:
Jup, will do. Just do double check: @weicaiyang, we currently do not have a top-level test that checks this, right?

Reply:
yeah I just added that one recently (due to this testpoint). thanks for confirming, I will open an issue so that we can add or enhance a test for this.

Reply:
SG, thanks Michael

Reply:
Filed an issue here: #15272.